notebooks/02_merge_heterogeneous_datasets_for_detection.ipynb

```diff
@@ -1,16 +1,17 @@
 {
  "cells": [
   {
+   "attachments": {},
    "cell_type": "markdown",
    "id": "0c0b77ed",
    "metadata": {},
    "source": [
     "### Merge Heterogeneous Datasets for Detection\n",
```

Comment:
```suggestion
    "### Merge Heterogeneous Datasets for Detection\n",
    "[notebook file link](https://github.com/openvinotoolkit/datumaro/blob/develop/notebooks/02_merge_heterogeneous_datasets_for_detection.ipynb)\n"
```
Would you add an access link or badge to the notebook files itself to the header of every notebook example? There is no way to access the notebook example file (`*.ipynb`) in the [documentation](https://openvinotoolkit.github.io/datumaro/stable/docs/jupyter_notebook_examples/notebooks/01_merge_multiple_datasets_for_classification.html).

Reply:
Reflected in 7e11be7d13354f76c64329077ecad65d5a33368a, thank you for good suggestion!